src/app/modals/encryptwallet/encryptwallet.component.ts

```diff
@@ -35,34 +36,32 @@ export class EncryptwalletComponent {
       this.log.d(`check password equality: ${password.password === this.password}`);
 
       if (this.password === password.password) {
-        this._rpc.state.set('ui:spinner', true);
+        this._rpcState.set('ui:spinner', true);
```

Comment:
This is ui state, not rpcState

Reply:
I'm deleting every ui state from the rpcState
#769